itwin-platform-access/imodels-access-frontend/package.json

```diff
@@ -52,8 +52,8 @@
     "typescript": "~4.4.0"
   },
   "peerDependencies": {
-    "@itwin/core-bentley": "^3.6.0",
-    "@itwin/core-common": "^3.6.0",
-    "@itwin/core-frontend": "^3.6.0"
+    "@itwin/core-bentley": "^3.6.0 || ^4.0.0",
```

Comment:
what about the actual clients?

Reply:
those are the only package.json that have peer dependency sections

Reply:
could you update the object-storage deps from 1.3 -> 1.6 ?

Reply:
dont add peer deps, update the direct deps

Reply:
is this the change you were loooking for or only the direct deps, not dev dependencies?

Reply:
no. revert the dev + direct deps doing ||

Reply:
no; please look at above comment